rs/tests/BUILD.bazel

```diff
@@ -143,6 +143,107 @@ uvm_config_image(
     tags = ["manual"],  # this target will be built if required as a dependency of another target
 )
 
+# Packages we install into the image; see base image definition and its
+# 'manifest = ...' field for more information
+PACKAGES_NOBLE = [
+    "@noble//bash/amd64",
+    "@noble//ca-certificates/amd64",
+    "@noble//coreutils/amd64",
+    "@noble//dosfstools/amd64",
+    "@noble//gawk/amd64",
+    "@noble//gzip/amd64",
+    "@noble//libssl3t64/amd64",
+    "@noble//libunwind8/amd64",
+    "@noble//mtools/amd64",
+    "@noble//openssh-client/amd64",
+    "@noble//rsync/amd64",
+    "@noble//tar/amd64",
+    "@noble//zstd/amd64",
+]
+
+tar(
+    name = "sh_noble",
+    mtree = [
+        # needed as dpkg assumes sh is installed in a typical debian installation.
+        "./usr/bin/sh type=link link=/bin/bash",
+    ],
+)
+
+tar(
+    name = "mkfsvfat_noble",
+    mtree = [
+        # symlink instead of updating the PATH
+        "./usr/bin/mkfs.vfat type=link link=/sbin/mkfs.vfat",
+    ],
+)
+
+tar(
+    name = "awk_noble",
+    mtree = [
+        # we install the GNU variant but call 'awk' so we symlink
+        "./usr/bin/awk type=link link=/usr/bin/gawk",
+    ],
+)
+
+# Creates /var/lib/dpkg/status with installed package information.
+dpkg_status(
+    name = "dpkg_status_noble",
+    controls = [
+        "%s:control" % package
+        for package in PACKAGES_NOBLE
+    ],
+)
+
+passwd(
+    name = "passwd_noble",
+    entries = [
+        dict(
+            gecos = ["root"],
+            gid = 0,
+            home = "/root",
+            shell = "/usr/bin/bash",
+            uid = 0,
+            username = "root",
+        ),
+    ],
+)
+
+oci_image(
+    name = "ubuntu_noble_test_runtime_image_src",
+    architecture = "amd64",
+    os = "linux",
+    target_compatible_with = [
+        "@platforms//os:linux",
+    ],
+    tars = [
+        ":dpkg_status_noble",
+        ":passwd_noble",  # needed because we ssh in
+
+        # symlinks
+        ":awk_noble",
+        ":mkfsvfat_noble",
+        ":sh_noble",
+    ] + PACKAGES_NOBLE,
+)
+
+oci_tar(
+    name = "ubuntu_noble_test_runtime.tar",
+    image = ":ubuntu_noble_test_runtime_image_src",
+    repo_tags = ["ubuntu_noble_test_runtime:image"],
+)
+
+uvm_config_image(
+    name = "colocate_uvm_noble_config_image",
```

Comment:
Where is this UVM config image being used? Or is that planned for some later PR?

Reply:
I'll be using it in something closer to https://github.com/dfinity/ic/pull/939, then we can remove the `focal` variants. The plan is to come at it in parts just to keep the changes down.